### ISSUE TITLE
Properly formatted 'funannotate predict' files do no exist in this directory

### ISSUE THREAD

**Issue by aldendirks** [nextgenusfs/funannotate#961]
I don't know if this counts as a bug, but wanted to share that I attempted to update an annotated genome with the NCBI locus tag and SBT file. I didn't know the best way to do this so I started with `funannotate update` but got the error `Properly formatted 'funannotate predict' files do no exist in this directory`. I then tried updating the annotated genome with `funannotate annotate` but got the same error. I then deleted the `update` directories and it worked (after first trying to delete the `annotatation_results` directory). Anyway, not sure how you would recommend to update an annotated file with locus tag and SBT but documentation on how to do so could be helpful. 

**What command did you issue?**
funannotate update -i fun_out --name $LOCUS_TAG RI367 --sbt $SBT_FILE 
funannotate annotate -i fun_out --antismash $ANTISMASH_GBK_PATH --cpus $CORES --tmpdir $TMPDIR --sbt $SBT_FILE --rename RI367

**Logfiles**
[Sep 14 12:00 PM]: OS: Red Hat Enterprise Linux 8.6, 36 cores, ~ 196 GB RAM. Python: 3.8.15
[Sep 14 12:00 PM]: Running 1.8.13
[Sep 14 12:00 PM]: Properly formatted 'funannotate predict' files do no exist in this directory

**Comment by hyphaltip** [comment]
did you run funannotate 'predict' already and there are no errors with that?

**Comment by aldendirks** [comment]
Right, no errors with funannotate predict, and no errors with annotate. The genome is annotated, all good. It was just in trying to rename the files with the locus tag and add the SBT file when I tried using funannotate update that I got that error. It worked with funannotate annotate, once I deleted the update directories. So no issue really, just wanted to bring that to your attention since it was odd.

**Comment by peritob** [comment]
Just adding a comment here as I had this same error using this script on successful prediction files and independently run interproscan .xml output placed in results/annotate_misc/ directory. Using  OS: CentOS 6.9, 32 cores, ~ 132 GB RAM. Python: 3.8.2. Running funannotate version 1.8.15 locally installed with dependencies on HPC running PBS-Pro.


`funannotate annotate -i ${outdir}/predict_results/ --sbt $wdir/${sbt_name} --busco_db embryophyta --species ${prefix_name} --iprscan ${outdir}/annotate_misc/${prefix_name}.xml -o ${outdir} --cpus 24`

I tried multiple variations but this alternative script worked. 


`funannotate annotate --gff ${outdir}/predict_results/${prefix_name}.gff3 --fasta ${outdir}/predict_results/${prefix_name}.scaffolds.fa --sbt $wdir/${sbt_name} --busco_db embryophyta --species ${prefix_name} --iprscan ${outdir}/annotate_misc/${prefix_name}.xml -o ${outdir} --cpus 24
`
